wuzz.go

```diff
@@ -149,7 +133,12 @@ func (e *ViewEditor) Edit(v *gocui.View, key gocui.Key, ch rune, mod gocui.Modif
 		e.backTabEscape = true
 		return
 	}
-
+	if key == gocui.KeyHome {
+		_ = e.GoToHome(v)
```

Comment:
@Eraden there is no such thing "backtab" or "shift+tab" in terminal, if you press shift+tab it generates the sequence `\033]Z`. The purpose of this function is to catch these character sequences. `backTabEscape` is a boolean state flag, which is `true` if the previous key was the first part of the "backtab" sequence.